sageopt/tests/test_symbolic/test_polynomials.py

```diff
@@ -185,7 +242,8 @@ def test_infeasible_poly_domain(self):
         except RuntimeError as err:
             err_str = str(err)
             assert 'seem to be infeasible' in err_str
-        pass
+        
+        # self.assertRaises(RuntimeError, PolyDomain.__init__, 1, logspace_cons=cons, log_AbK=(A, b, K))
```

Comment:
What are these commented out lines from?

Reply:
The line was meant to cover line 569-570 of sageopt/sageopt/symbolic/polynomials.py. Would that line correctly ensure that the keyword arguments "log_AbK" and "logspace_cons" are mutually exclusive?

Reply:
I believe these lines would serve to test that case. If you un-comment those lines I'll be able to merge this in! (For good measure, pinging @HC-HC.)